receptor_affinity/debugger.py

```diff
@@ -13,34 +13,65 @@
 
 
 class DiagNode:
+    """
+    A DiagNode takes a datapath, a node_id and a listener port and spins up an instance of a
+    receptor node, exposing a few of the internal functions of receptor via an API so that a user
+    testing a mesh can gain an insight into some of the internals.
+
+    The three arguments are the same arguments one would pass to receptor itself.
+
+    data_path: points to the data path where the manifest and message store will reside
+    listen: defines the url that the receptor server will start on
+    node_id: declares the name of the node, or the node_id
+
+    The API exposes 4 endpoints:
```

Comment:
Which API exposes four endpoints? The receptor process' socket address as defined by the `listen` parameter?

Reply:
Your're right............*sigh*